Makefile

```diff
@@ -148,9 +149,9 @@ ko-sget:
 .PHONY: ko-local
 ko-local:
 	$(create_kocache_path)
-	KO_DOCKER_REPO=ko.local LDFLAGS="$(LDFLAGS)" GIT_HASH=$(GIT_HASH) GIT_VERSION=$(GIT_VERSION) \
+	LDFLAGS="$(LDFLAGS)" GIT_HASH=$(GIT_HASH) GIT_VERSION=$(GIT_VERSION) \
 	KOCACHE=$(KOCACHE_PATH) ko build --base-import-paths \
-		--tags $(GIT_VERSION) --tags $(GIT_HASH) \
+		--tags $(GIT_VERSION) --tags $(GIT_HASH) --local \
```

Comment:
Please revert this change, we export the `KO_DOCKER_REPO` in the top of the makefile and then we need to set this here to build locally, the `KO_DOCKER_REPO` have more priority over the `--local`

Reply:
done